src/streamlink/plugins/mediavitrina.py

```diff
@@ -1,4 +1,5 @@
 """
+$description Russian live TV channels from CTC Group, including Che, CTC, CTC Love and Domashny.
```

Comment:
there is more than `CTC Group`, description should be like [cdnbg.py](https://github.com/streamlink/streamlink/pull/4391/files#diff-e7158d7c7de978893a186eacead94ed6f36f018eacf706dc19f91456c862a831)